Automattic/wp-calypso
# Subscribers form: integrate record tracks events

#### Proposed Changes

* Integrated record tracks events on the Subscribers form.
* Events:
  * calypso_subscriber_add_form_display
  * calypso_subscriber_add_form_individual_subscribers
  * calypso_subscriber_add_form_csv_subscribers
  * calypso_subscriber_add_form_error
  * calypso_subscriber_add_form_success

#### Testing Instructions

* Go to `/setup/subscribers?flow=newsletter&complete-setup=true&siteSlug={SLUG}`
* Check if add subscriber form works as usual

#### Pre-merge Checklist






Related to #
